server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceShardsAllocator.java

```diff
@@ -163,6 +169,7 @@ public String toString() {
                 onNoLongerMaster();
             }
         });
+        this.clusterService = clusterService;
```

Comment:
I think this is not used. Was it added in a prior versions and no longer needed?

Reply:
:+1: see baa1c5b8bb8a260bbed79b52d0cdb0badfd10f4a